### DISCUSSION TITLE
How to set up a shared model directory with SD?

### DISCUSSION THREAD

**Discussion by guge2772135136** [lllyasviel/Fooocus#236] [Q&A] [answered]
The impact can be achieved by modifying the configuration file. May I ask which configuration file each of you guides to modify?

**Comment by barepixels** [comment, marked as answer]
You have 2 choice

1) hard code modules/path.py
but you have to make the same change every time lllyasviel/Fooocus get update

2) Use https://github.com/MoonRide303/Fooocus-MRE that allows you save models path in the paths.json

**Comment by Alcyon6** [comment]
I use symlinks. Used to make them on the command line but recently learned how to do with with Ctrl + Shift dragging.

**Comment by barepixels** [comment]
From my experience symlinks both /D and /J options doesn't work well with Fooocus.  Everytime Fooocus gets update the symlinks gets replace with actual folders.  Windows 10 here

See this discussion  https://github.com/MoonRide303/Fooocus-MRE/issues/12
